### PR TITLE
Add a new pack for HIPAA

### PR DESCRIPTION
**Pull request by altmannyarden**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/25126

## Description
Created a new pack for HIPAA.
The pack contains:
- new playbook
- Incident type
- Incident fields
- Incident layouts
.
## Screenshots
image

## Minimum version of Demisto
- [ ] 4.5.0
- [x] 5.0.0
- [ ] 5.5.0

### CODE REVIEW

**Comment by altmannyarden** [comment]
@yaron-libman Hi, Please check the playbook yml and the pack metadata.

**Comment by kirbles19** [comment]
@michalgold @altmannyarden 

We won't have time to review this PB before code freeze. We can either release as is or save it for next release, your call.

**Comment by yaron-libman** [comment]
Reviewed

File: Packs/HIPAA-BreachNotification/pack_metadata.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    "description": "HIPAA (Health Insurance Portability and Accountability Act) is a binding regulation for organizations operating in the USA. This pack follows through with the notification procedures in case of PHI breach according to HIPAA requirements. The pack contains a playbook, incident type, layouts and incident fields. HIPAA breach notification incident and follows through with the notification procedures.",
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "created": "2020-06-03T08:25:53Z",
+    "categories": [],
+    "tags": [],
+    "useCases": [],
```
Comment (michalgold) [inline Packs/HIPAA-BreachNotification/pack_metadata.json:12]: Use case: HIPPA Breach Notification, Compliance
Tags: HIPPA

**Pull request merged**